pkg/didcomm/protocol/introduce/service.go

```diff
@@ -421,13 +440,13 @@ func (s *Service) handle(msg *metaData, dest *service.Destination) error {
 		)
 
 		if dest != nil {
-			followup, err = next.ExecuteOutbound(msg, dest)
+			followup, err = next.ExecuteOutbound(s.ctx, msg, dest)
```

Comment:
Context is supposed to be defined for a particular request/transaction - not as a member variable used across transactions. See prior comment.

Reply:
it is internal context

Reply:
the same logic in did-exchange

Reply:
internal context is ok.